tmpl/layout.tmpl

```diff
@@ -28,7 +28,7 @@
 <br class="clear">
 
 <footer>
-    © <?js= new Date().getUTCFullYear() ?> <a href="http://www.openfin.co" target="_blank">OpenFin</a> All Rights Reserved.
+    © <?js= new Date().getUTCFullYear() ?> <a href="http://github.com/fin-hypergrid" target="_blank">The Hypergrid Authors</a>. All Rights Reserved.
```

Comment:
@joneit We want the license to be `© OpenFin and The Hypergrid Authors 2015-2018`